target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -1,35 +1,33 @@
+#!/bin/sh
+
```

Comment:
These are non-executable lib files. There should be no shebang.

Reply:
thanks, got rid of shebang